### PR TITLE
Prevent confusing errors from leaking into the general logs

### PR DESCRIPTION
**Pull request by originalsouth**
### Changes
Prevent confusing errors from leaking into the general logs

---

### Code Checklist
- [x] All the commits in this PR are properly PGP-signed and verified.
- [ ] This PR only contains functionality relevant to the issue; tickets have been created for newly discovered issues.
- [ ] I have written unit tests for the changes or fixes I made.
- [x] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [x] I have performed a self-review of my code and refactored it to the best of my abilities.

### Communication
- [x] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [x] I have made corresponding changes to the documentation, if necessary.
- [x] I have included comments in the code to elaborate on what is not self-evident from the code itself, including references to issues and discussions online, or implicit behavior of an interface.

---
## Checklist for code reviewers:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---
## Checklist for QA:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

File: octopoes/octopoes/xtdb/client.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ def _verify_response(response: Response) -> None:
         try:
             response.raise_for_status()
         except HTTPStatusError as e:
-            logger.error(e.response.status_code)
+            logger.debug(e)
             if e.response.status_code != codes.NOT_FOUND:
                 logger.error(response.request.url)
```
Comment (dekkers) [inline octopoes/octopoes/xtdb/client.py:64]: I think we still want to log with level error when the status code isn't 404? Else we just log the request url en content if debug isn't enabled which isn't very useful.

Comment (originalsouth) [reply octopoes/octopoes/xtdb/client.py:64]: This was the previous behavior https://github.com/minvws/nl-kat-coordination/pull/2795/files#diff-1f4c0c32e0407dc2b0e94a92c5bbd9979d8a810c4d980a10769be9f2ad49565b. The logging was actually useful for creating that patch -- to understand all the various ways that errors can occur.

Either is fine with me.

Comment (ammar92) [reply octopoes/octopoes/xtdb/client.py:64]: Agree with both; logging the entire response context isn't appropriate in this case, and logging the status error (code or code/ reason combination) is more meaningful (and preferably with some text such that we can differentiate between request/ response logs)

Comment (originalsouth) [reply octopoes/octopoes/xtdb/client.py:63]: Another try 47194be7a5ad76861245623ebb914c72a43f0a7c

Comment (originalsouth) [reply octopoes/octopoes/xtdb/client.py:64]: Another try in a22ef2a7ad9770c44b3dfa757fc768effe88ac31

Comment (originalsouth) [reply octopoes/octopoes/xtdb/client.py]: Resolved in 8186b8ceb596dd36b96954424dbc76a03a5f9e67

**Review by underdarknl — APPROVED** [review]

**Pull request merged**